README.md

```diff
@@ -660,22 +657,9 @@ nock('http://zombo.com').get('/').thrice().reply(200, 'Ok')
 
 To repeat this response for as long as nock is active, use [.persist()](#persist).
 
-### Delay the response body
-
-You are able to specify the number of milliseconds that the response body should be delayed. Response header will be replied immediately.
-`delayBody(1000)` is equivalent to `delay({body: 1000})`.
-
-```js
-nock('http://my.server.com')
-  .get('/')
-  .delayBody(2000) // 2 seconds
-  .reply(200, '<html></html>')
-```
-
-NOTE: the [`'response'`](http://nodejs.org/api/http.html#http_event_response) event will occur immediately, but the [IncomingMessage](http://nodejs.org/api/http.html#http_http_incomingmessage) will not emit its `'end'` event until after the delay.
-
 ### Delay the response
 
+Nock can simulate response latency to allow you to test timeouts, race conditions, an other timing related scenarios.  
```

Comment:
I'm not sure this is completely accurate, but I'd suggest something like this:

> Nock can simulate the effect of response latency to allow you to test timeouts, race conditions, an other timing related scenarios. Rather than waiting for real clock time to pass it compares the requested delay to the timeout parameter passed to the overridden request methods (i.e. `http.request()` and `http.ClientRequest#setTimeout()`) and emits the timeouts almost immediately. This allows you to test timeouts without using fake timers or slowing down your tests.

Reply:
The added detail here is only true when delaying the connection and a timeout is set on the request. So while I'm good with the added verbiage, I think it should go below where the details of `delayConnection` are covered. 
I was trying to keep this opening section on delays high-level and only include the info the top 90% of users are going to want.